### PR TITLE
extract locale config into yml

### PR DESCRIPTION
**Pull request by s01us**
in creating a node script for updating the locales it's become clear our locales config is based into some ruby.

I've extracted them into yaml, so that they are : 

- easier for node to read 
- easier for users to find and edit

### CODE REVIEW

**Comment by robguthrie** [comment]
FYI - we're on ruby 2.2 now, which garbage collects symbols... So not need to have the silly string stuff in places, you can convert to symbols as soon as you like and treat everything as symbols.

**Comment by s01us** [comment]
sweet, I'll see what I can do. 
yaml doesn't really do symbol, so we'll still have some string being pulled in

File: lib/loomio_i18n.rb [hunk 1]
```diff
@@ -1,29 +1,23 @@
 module Loomio
-  module I18n
+  class I18n
+    require 'yaml'
+    config = YAML.load_file("config/loomio_i18n.yml")['loomio_i18n']
 
-    SELECTABLE_LOCALES = %i( en an be-BY bg-BG ca cs zh-TW da de eo es el fr id it he hu ja ko ml nl-NL pt-BR ro sr sr-RS sk sv vi tr uk )
+    def self.symbolize(object)
```
Comment (gdpelican) [inline lib/loomio_i18n.rb]: Not totally sure you need this method to symbolize everything.

Would something like this do the job?

```
module Loomio
  class I18n
    require 'yaml'
    config = YAML.load_file("config/loomio_i18n.yml")['loomio_i18n']

    SELECTABLE_LOCALES = Array(config['selectable_locales']).map(&:to_sym)
    DETECTABLE_LOCALES = Array(config['additional_detectable_locales']).map(&:to_sym) + SELECTABLE_LOCALES

    RTL_LOCALES  = Array(config['rtl_locales']).map(&:to_sym)
    TEST_LOCALES = Array(config['test_locales']).map(&:to_sym)

    FALLBACKS = Hash(config['fallbacks']).with_indifferent_access

  end
end
```

**Comment by gdpelican** [comment]
Looks pretty great to me. I'll merge once the build's done.

**Pull request merged**